alfresco-transform-libreoffice/alfresco-transform-libreoffice-boot/src/main/resources/application-default.yaml

```diff
@@ -2,4 +2,7 @@ queue:
   engineRequestQueue: ${TRANSFORM_ENGINE_REQUEST_QUEUE:org.alfresco.transform.engine.libreoffice.acs}
 transform:
   config:
-    location: classpath:libreoffice_engine_config.json
\ No newline at end of file
+    location: classpath:libreoffice_engine_config.json
+libreoffice:
```

Comment:
We should standardise these properties, this goes for the transform.config.location property as well. Jan suggests transform.core.imagemagick.prop1, So i guess we could have: 
**transform.config.location** > **transform.core.config.location** 
**libreoffice.executor.path** > **transform.core.libreoffice.executor.path**

@montgolfiere @KikoUK  - What do you think?

Reply:
Do you mean to have them under the same props tree transform.core..? I suggested that yesterday and I think you mentioned to keep them separate for now? I prefer the suggestion above, having them together.

Reply:
I prefer a well-known prefix - but not too long, eg. do we actually need "executor", eg.

* transform.core.libreoffice.path